examples/databases/mariadb/devbox.json

```diff
@@ -9,6 +9,8 @@
         "mysql -u devbox_user -p -D devbox_lamp"
       ],
       "test_db_setup": [
+        "mkdir -p /tmp/devbox/mariadb/run",
+        "export MYSQL_UNIX_PORT=/tmp/devbox/mariadb/run/mysql.sock",
```

Comment:
love it

Reply:
I had to hold my nose on this to unblock Lucille, because these examples should show the ideal code a user would write, and this is a workaround we're implementing for our specific testscripts CICD tests.